src/P/EitherT.hs

```diff
@@ -1,5 +1,12 @@
 module P.EitherT (
-    firstEitherT
+    EitherT(..)
+  , eitherT
+  , bimapEitherT
+  , left
+  , hoistEither
+  , mapEitherT
```

Comment:
You can use module alias with an import whitelist if you want (we do elsewhere).